tutorials/text/text_intro.py

```diff
@@ -31,36 +32,38 @@
 ===================
 
 The following commands are used to create text in the pyplot
-interface
+interface and the object-oriented API:
 
-* :func:`~matplotlib.pyplot.text` - add text at an arbitrary location to the ``Axes``;
-  :meth:`matplotlib.axes.Axes.text` in the API.
+* `pyplot.text` - add text at an arbitrary location to the `Axes`;
+  `Axes.text` in the API.
```

Comment:
@anntzer  or others who understand: I am still not getting the sphinx auto-resolution.  What is the proper way to make these links?  Do I have to do `matplotlib.Axes.text`?  

See: https://3893-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/tutorials/text/text_intro.html#sphx-glr-tutorials-text-text-intro-py

Reply:
```
~.Axes.text
```
or
```
.Axes.text
```
should work, I think(?) (the leading dot means "search in other namespaces"; the tilde means "don't write the prefix in the rendered docs http://www.sphinx-doc.org/en/stable/domains.html#role-py:obj).

Reply:
Ok thanks a lot.  Hopefully that makes it into @dstansby doc on docs.